packages/transaction-controller/src/TransactionController.ts

```diff
@@ -342,6 +383,22 @@ export class TransactionController extends BaseController<
     this.getSelectedAddress = getSelectedAddress;
     this.securityProviderRequest = securityProviderRequest;
 
+    this.afterSign = hooks?.afterSign ? hooks.afterSign : () => true;
```

Comment:
Minor, but does the linter allow us to use `??` here?

That seems logical if it's happy with the longer ternary equivalent.

Reply:
Changed in both places `PendingTransactionTracker` and `TransactionController`.